catchpoint-mixin/panels.libsonnet

```diff
@@ -0,0 +1,418 @@
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true')
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topAvgContentLoadingTimeTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average content loading time',
+          targets=[t.topAvgContentLoadingTimeTestName],
+          description='The top average content loading time among all test names over the specified interval.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true')
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topAvgContentLoadingTimeNodeName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average content loading time by nodes',
+          targets=[t.topAvgContentLoadingTimeNodeName],
+          description='The top average content loading time among all node names over the specified interval.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true')
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      topAvgRedirectsTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average redirects by tests',
+          targets=[t.topAvgRedirectsTestName],
+          description='The top average number of redirects among all test names over the specified interval.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true'),
+
+      topAvgRedirectsNodeName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top average redirects by nodes',
+          targets=[t.topAvgRedirectsNodeName],
+          description='The top average number of redirects among all node names over the specified interval.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true'),
+
+      alertsPanel:
+        alertList.new('Catchpoint alerts')
+        + alertList.options.UnifiedAlertListOptions.withAlertInstanceLabelFilter(this.grafana.variables.queriesGroupSelectorAdvanced),
+
+      topErrorsByTestName:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top errors by tests',
+          targets=[t.topErrorsByTestName],
+          description='The top number of errors encountered among all test names over the specified interval.'
+        )
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true'),
+
+      // Web Performance by Tests Dashboard Panels
+      pageCompletionTime:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Page completion time',
+          targets=[t.pageCompletionTime, t.pageTotalLoadTime],
+          description='Time taken for the browser to fully render the page after all resources are downloaded.'
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ms')
+        + g.panel.timeSeries.fieldConfig.defaults.custom.withSpanNulls('true'),
```

Comment:
This panel also suffers from uh, legend congestion shall we say 
![image](https://github.com/grafana/jsonnet-libs/assets/1282515/5aceaedb-8463-4d8f-ba09-ec31eade4fca)